presentation-api/controlling-ua/reconnectToPresentation_error_manualTest.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
s/_manualTest/-manual/ in file name. See

http://testthewebforward.org/docs/manual-test.html